ui/raidboss/data/06-ew/raid/p4n.ts

```diff
@@ -0,0 +1,198 @@
+// import { callOverlayHandler } from '../../../../../resources/overlay_plugin_api';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { PluginCombatantState } from '../../../../../types/event';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export interface Data extends RaidbossData {
+  bodyActor?: PluginCombatantState;
+}
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.AsphodelosTheFourthCircle,
+  triggers: [
+    {
+      id: 'P4N Elegant Evisceration',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A50', source: 'Hesperos' }),
+      response: Responses.tankCleave(),
+    },
+    // Strong proximity Aoe
+    {
+      id: 'P4N Levinstrike Pinax',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A3F', source: 'Hesperos', capture: false }),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Go to Corners',
+        },
+      },
+    },
+    // Knockback from middle
+    {
+      id: 'P4N Well Pinax',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A3E', source: 'Hesperos', capture: false }),
+      delaySeconds: 4,
+      response: Responses.knockback(),
+    },
+    {
+      id: 'P4N Acid Pinax',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A3C', source: 'Hesperos', capture: false }),
+      response: Responses.spread(),
+    },
+    {
+      id: 'P4N Lava Pinax',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A3D', source: 'Hesperos', capture: false }),
+      response: Responses.stackMarker(),
+    },
+    {
+      id: 'P4N Decollation',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A51', source: 'Hesperos', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P4N Bloodrake',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A40', source: 'Hesperos', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P4N Hell Skewer',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A4F', source: 'Hesperos', capture: false }),
+      response: Responses.awayFromFront(),
+    },
+    {
+      id: 'P4N Belone Coils',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '69DD', source: 'Hesperos', capture: false }),
+      delaySeconds: 2,
+      alarmText: (data, _matches, output) => {
+        if (data.role === 'tank')
+          return output.avoidTankTower!();
+        if (data.role === 'dps')
+          return output.avoidDpsTower!();
+        if (data.role === 'healer')
+          return output.avoidHealerTower!();
+      },
+      outputStrings: {
+        avoidTankTower: {
+          en: 'Avoid Tank Tower',
```

Comment:
If possible, can this be written positively rather than negatively? (I think it's not just "avoid tank tower", it's "get other towers").  I think even "Get Other Role Tower" might be better?

Reply:
Damn he suggested that and I was against it. The more you know ;)